components/select_team/components/select_team_item.jsx

```diff
@@ -59,14 +58,13 @@ export default class SelectTeamItem extends React.PureComponent {
         return (
             <div className='signup-team-dir'>
                 {showDescriptionTooltip}
-                <Link
-                    to={`/${this.props.team.name}/channels/town-square`}
+                <a
```

Comment:
a `<a>` tag should always have a `href`

And are you sure this the right call? I thought team switches etc were handled by router changes... I am not sure what `this.props.onTeamClick` does. I am 0/5 as long as you are confident...

Reply:
I'm not 100% sure about how mandatory is that. Here, the onTeamClick what does is try to join to the team, and if everything goes well, then, redirect to the new "route", if you directly write the route, will get you into the channel, but if something goes wrong, it doesn't show any error.

Reply:
Without a `href` a link is treated as a placeholder link by the browser. Which means the default CSS is slightly different and hover actions are slightly different. Also bootstrap classes will sometimes fail. It is more a CSS issue.

I would suggest setting `href="#"` as that is what we do everywhere else.

I, personally, prefer `href="javascript:void(0)"` but that is not the mattermost way. ;)

Reply:
Done :)